docs/stable/1-client.md

```diff
@@ -190,30 +200,30 @@ group of peers the official Status client uses. The others are test networks.
 
 #### Mobile nodes
 
-This is a Whisper node which connects to part of the Whisper network. It MAY
-relay messages. See next section for more details on how to use Whisper to
+This is a Waku node which connects to part of the Waku network. It MAY
+relay messages. See next section for more details on how to use Waku to
 communicate with other Status nodes.
 
-### Transport privacy and Whisper usage
+### Transport privacy and Waku usage
 
-Once a Whisper node is up and running there are some specific settings required
+Once a Waku node is up and running there are some specific settings required
 to commmunicate with other Status nodes.
 
-See [3/WHISPER-USAGE](https://specs.status.im/spec/3) for more details.
+See [3/WAKU-USAGE](https://specs.status.im/spec/3) for more details.
```

Comment:
This spec doesnt exist

Reply:
My presumption is that this will reference https://github.com/status-im/specs/blob/master/docs/stable/3-whisper-usage.md, after the whole rewrite has completed.

Reply:
I'm going to revert this change, as per https://github.com/status-im/specs/pull/114#issuecomment-625894903